app/js/app-menu.js

```diff
@@ -42,6 +45,101 @@ module.exports = function () {
     return chiseInstance.loadSample(filename, 'app/samples/');
   }
 
+  appMenu.launchWithModelFile = function() {
+    var url_path = getParameterByName('url');
+    var uri_path = getParameterByName('uri');
+    var chiseInstance = appUtilities.getActiveChiseInstance();
+    if(url_path != undefined)
+      loadFromURL(url_path, chiseInstance);
+    else if(uri_path != undefined)
+      loadFromURI(uri_path, chiseInstance);
+
+  }
+
+  function loadFromURL(filepath, chiseInstance){
+    var loadCallbackSBGNMLValidity = function (text) {
+      validateSBGNML(text);
+    }      
+    var loadCallbackInvalidityWarning  = function () {
+      new BackpromptInvalidFileView.render();
```

Comment:
@ahmetcandiroglu where is `BackpromptInvalidFileView` defined?

Reply:
@kinimesi it should be `promptInvalidFileView.render()` and it is defined in `app-menu.js`.